extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -131,10 +131,10 @@ public boolean isAnyPropertySet() {
                 sqlLoadScript.isPresent() ||
                 batchFetchSize > 0 ||
                 statistics ||
-                query.isAnyPropertySet() ||
-                database.isAnyPropertySet() ||
-                jdbc.isAnyPropertySet() ||
-                log.isAnyPropertySet() ||
+                query.isPresent() ||
+                database.isPresent() ||
+                jdbc.isPresent() ||
+                log.isPresent() ||
```

Comment:
I suppose the next step would be to see if we can make some of the properties in the optional groups be individually non-optional.